### PR TITLE
[ServiceBus] Bump uamqp dependency

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,19 @@
...
 **New Features**
 
 * Updated `forward_to` and `forward_dead_lettered_messages_to` parameters in `create_queue`, `update_queue`, `create_subscription`, and `update_subscription` methods on sync and async `ServiceBusAdministrationClient` to accept entities as well, rather than only full paths. In the case that an entity is passed in, it is assumed that the entity exists within the same namespace used for constructing the `ServiceBusAdministrationClient`.
 
+**Bug Fixes**
+
+* Updated uAMQP dependency to 1.3.0.
+  - Fixed bug that sending message of large size triggering segmentation fault when the underlying socket connection is lost (#13739, #14543).
+  - Fixed bug in link flow control where link credit and delivery count should be calculated based on per message instead of per transfer frame (#16934).
+  - Fixed memory leaks in the process of link attach where source and target cython objects are not properly deallocated (#15747).
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:16]:
```suggestion
```

File: sdk/servicebus/tests.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: servicebus
+      InjectedPackages: $(WheelLocationsList)
```
Comment (yunhaoling) [inline sdk/servicebus/tests.yml:7]:
```suggestion
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,18 @@
 
 ## 7.1.1 (Unreleased)
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: should we add the date here or wait on that until tomorrow?

**Review by swathipil — APPROVED** [review]

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,18 @@
...
 
+This version and all future versions will require Python 2.7 or Python 3.6+, Python 3.5 is no longer supported.
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:5]: I believe this comment has already been added to [this PR](https://github.com/Azure/azure-sdk-for-python/pull/17751)?

File: sdk/servicebus/azure-servicebus/CHANGELOG.md (hunk 3 above)
Comment (yunhaoling) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: will merge first and resolve in your PR :P

File: sdk/servicebus/azure-servicebus/CHANGELOG.md (hunk 4 above)
Comment (yunhaoling) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:5]: will merge first and resolve in your PR :P

**Pull request merged**